### PR TITLE
Color: Add Brand color palette

### PR DESCRIPTION
**Pull request by ayeshakmaz**
### Summary
Add documentation for the brand colors

### Links

- [Jira](link to Jira ticket(s))
- [TDD](link to Paper doc)
- [Figma](link to Figma file)

### Checklist

- [ ] Added unit and Flow Tests
- [ ] Added documentation + accessibility tests
- [ ] Verified accessibility: keyboard & screen reader interaction
- [ ] Checked dark mode, responsiveness, and right-to-left support
- [ ] Checked stakeholder feedback (e.g. Gestalt designers)

### CODE REVIEW

**Review by AlbertCarreras — APPROVED** [review]
LGTM

File: docs/pages/color.js [hunk 1]
```diff
@@ -5,9 +5,26 @@ import MainSection from '../components/MainSection.js';
 import PageHeader from '../components/PageHeader.js';
 import CardPage from '../components/CardPage.js';
 import ColorPalette from '../components/ColorPalette.js';
+import ColorTile from '../components/ColorTile.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
+const colors = [
+  { 'name': 'Pushpin', 'id': 'red', 'textColor': 'white' },
```
Comment (dangerismycat) [inline docs/pages/color.js:13]: weird that Prettier isn't complaining about the unnecessary `'` in the keys…

**Comment by ponori** [comment]
@cintiapub Just added the copy change. Feel free to have a look if/when you have a chance.

**Comment by cintiapub** [comment]
> @cintiapub Just added the copy change. Feel free to have a look if/when you have a chance.

It looks good to me, @ponori. Thank you!

**Pull request merged**